server/sonar-server/src/main/java/org/sonar/server/organization/OrganizationCreationImpl.java

```diff
@@ -252,4 +254,10 @@ private void addCurrentUserToGroup(DbSession dbSession, GroupDto group, int crea
       dbSession,
       new UserGroupDto().setGroupId(group.getId()).setUserId(createUserId));
   }
+
+  private void insertOrganizationMember(DbSession dbSession, OrganizationDto organizationDto, UserDto userDto) {
```

Comment:
ok, I wasn't explicit enough apparently. If this class has the same responsibility and spirit than XUpdater classes, then it should be named accordingly. We don't need several names for the same thing.
I asked the question to confirm I wasn't missing something